avaje-config/src/main/java/io/avaje/config/ConfigurationLog.java

```diff
@@ -9,21 +9,7 @@
  * control how the events are logged. For example, it might delay logging messages
  * until logging implementation has finished configuration.
  */
-public interface ConfigurationLog {
-
-  /**
-   * Invoked when the configuration is being initialised.
-   */
-  default void preInitialisation() {
```

Comment:
Removing these methods is a breaking change. I don't understand why this is removed.  Prefer this to be in a separate PR with some justification for the change.